boards/arm/lpcxpresso54114/lpcxpresso54114.dts

```diff
@@ -17,7 +17,7 @@
 	};
 
 	chosen {
-		zephyr,sram = &sram1;
+		zephyr,sram = &sram0;
```

Comment:
add a comment in commit message about this change.

Reply:
Ok, modified commit message.